pkgs/applications/graphics/animbar/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchurl, cmake, qt4, file, gcc }:
+
+stdenv.mkDerivation rec {
+  pname = "animbar";
+  version = "1.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${name}.tar.bz2";
+    sha256 = "0836nwcpyfdrapyj3hbg3wh149ihc26pc78h01adpc7c0r7d9pr9";
+  };
+
+  nativeBuildInputs = [ cmake  ];
+
+  buildInputs = [ qt4 file ];
```

Comment:
Have you tried to build this with qt5? We are trying to get rid of qt4 in nixpkgs, see #33248 .

Reply:
Yes, I have tried. It fails to build without qt5.
Regarding removal of qt4: I've not involved with the discussion so far, but I'm in the camp of @vcunat of peaceful coexistence. I see no point in removing an entire toolkit from Nixpkgs 'just because you can'. Like in the case of this software: I'm not aware of a real equivalent. It does its (simple but unique) job. And it is uncertain whether it will ever be changed to some new version of Qt - what need would there be. If you don't have software that uses qt4 installed, it does not burden you. And if you want to use it, you are willing to pay the 'price' of extra storage. 
Of course, in case of software that can be built with both there is no need to have two versions - and moving those to "qt5 exclusive" is a sane thing to optimise the common experience....

Reply:
> Yes, I have tried. It fails to build without qt4.

Fine, that's all I wanted to know. My opionion on the general issue is that, if a package supports both qt4 and qt5, we should prefer qt5. But there's no reason to refuse packages that need qt4.